spec/spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 
 CodeClimate::TestReporter.start
 
+Dir[File.expand_path('spec/support/matchers/*.rb')].each{ |file| require file }
```

Comment:
Could create a meta file called `spec/support/matchers.rb` and include it. And keep the logic for loading the individual matchers in it...